Makefile

```diff
@@ -36,6 +36,10 @@ test: lint ## Run tests
 	pytest tests/scripts
 	shellcheck **/*.sh
 
+requirements-spark3.txt: export CUSTOM_COMPILE_COMMAND := make requirements-spark3.txt
```

Comment:
why is this showing up?

Reply:
I made a change to setup.py, so needed to update requirements-spark3.txt otherwise the python3.8 check wouldn't pass